os/arch/arm/src/armv7-a/Kconfig

```diff
@@ -169,3 +169,8 @@ config ARMV7A_DECODEFIQ
 		Select this option if your platform supports the function
 		arm_decodefiq().  This is used primarily to support secure TrustZone
 		interrupts received on the FIQ vector.
+
+config ARMV7A_TRUSTZONE
```

Comment:
When do we need the config?
Basically RTL8730E uses the trustzone, then should we enable this by default? or this is only for the example?

Reply:
This config is used in syscall.h, but it's not defined in Kconfig.
Because of this, when config is changed through menuconfig, build fails because CONFIG_SYS_RESERVED value is getting different.
```
#ifdef CONFIG_LIB_SYSCALL

#ifndef CONFIG_SYS_RESERVED
#error "CONFIG_SYS_RESERVED must be defined to have the value"
#endif //CONFIG_SYS_RESERVED

#ifdef CONFIG_BUILD_PROTECTED

#ifdef CONFIG_ARMV7A_TRUSTZONE
#if CONFIG_SYS_RESERVED != 11
#error "CONFIG_SYS_RESERVED must have the value 11"
#endif
#else
#if CONFIG_SYS_RESERVED != 8
#error "CONFIG_SYS_RESERVED must have the value 8"
#endif
#endif //CONFIG_ARMV7A_TRUSTZONE

#else //CONFIG_BUILD_PROTECTED

#ifdef CONFIG_ARMV7A_TRUSTZONE
#if CONFIG_SYS_RESERVED != 7
#error "CONFIG_SYS_RESERVED must have the value 7"
#endif
#else
#if CONFIG_SYS_RESERVED != 4
#error "CONFIG_SYS_RESERVED must have the value 4"
#endif
#endif //CONFIG_ARMV7A_TRUSTZONE

#endif //CONFIG_BUILD_PROTECTED

#endif //CONFIG_LIB_SYSCALL
```

Reply:
@sunghan-chang @ewoodev I removed ARMv7-A tz config.